doc/users/next_whats_new/polar-line-spans.rst

```diff
@@ -0,0 +1,5 @@
+``axhline`` and ``axhspan`` on polar axes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+... now draw circles and circular arcs (`~.axhline`) or full or partial annuli
+(`.axhspan`).
```

Comment:
Note that the references in this file are not resolved by sphinx (it finds both the Axes method and pyplot), and are causing the doc build to fail
 
```suggestion
... now draw circles and circular arcs (`~.Axes.axhline`) or full or partial annuli
(`.Axes.axhspan`).
```

Reply:
oops, fixed.